roles/openstack-setup/tasks/networks.yml

```diff
@@ -59,3 +63,22 @@
                             login_username=provider_admin
                             login_password={{ secrets.provider_admin_password }}
   with_items: neutron.router_interfaces
+  run_once: true
+
+- name: determine Neutron external interface name
+  neutron_network: name=external
+                   auth_url=https://{{ endpoints.main }}:5001/v2.0/
+                   login_tenant_name=admin
+                   login_username=provider_admin
+                   login_password={{ secrets.provider_admin_password }}
+  register: result
+  run_once: true
+
+- name: set neutron interface fact
+  set_fact: neutron_external_interface=brq{{ result.id|truncate(length=11,killwords=true,end='') }}
+
+- name: neutron external interface config
```

Comment:
should this be run on the network group rather than the controller group in cases where the neutron nodes are not running on the controllers ?